sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -735,12 +739,12 @@ public static Bound withoutValidation() {
      * {@link PCollection} of {@link TableRow TableRows} to a BigQuery table.
      */
     public static class Bound extends PTransform<PCollection<TableRow>, PDone> {
-      final TableReference table;
+      @Nullable final String jsonTableRef;
 
       final SerializableFunction<BoundedWindow, TableReference> tableRefFunction;
 
       // Table schema. The schema is required only if the table does not exist.
-      final TableSchema schema;
+      @Nullable final TableSchema schema;
```

Comment:
Presumably this also needs to become a `String` -- `TableSchema` is also not serializable.

And this also indicates a missing test?